convert/terraform/main.tf

```diff
@@ -0,0 +1,44 @@
+data "aws_caller_identity" "admin" {}
+data "aws_partition" "current" {}
+
+resource "aws_iam_role" "grandfathered_creation_role" {
+  name = var.grandfathered_creation_role
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Sid    = ""
+        Principal = {
+          AWS = "arn:${data.aws_partition.current.partition}:iam::${data.aws_caller_identity.admin.account_id}:root"
+        }
+      },
+    ]
+  })
+
+  lifecycle {
+    ignore_changes = [name, inline_policy]
+  }
+}
+
+module "domino_eks" {
+  source                       = "github.com/dominodatalab/terraform-aws-eks.git?ref=smooth-conversion-process"
+  deploy_id                    = var.deploy_id
+  region                       = var.region
+  number_of_azs                = var.number_of_azs
+  k8s_version                  = var.k8s_version
+  route53_hosted_zone_name     = var.route53_hosted_zone_name
+  eks_master_role_names        = var.eks_master_role_names
+  s3_force_destroy_on_deletion = true
```

Comment:
I guess this doesn't _really_ matter, but should this also come from the CDK config?

Reply:
Oops, yes, this was an oversight.

Reply:
Also does matter because we want this off in the real world. :)

Reply:
There is some slight ambiguity in how to handle this, since the config doesn't directly map, so I'll add this to the follow up PR.